hooks/emitter/emitter.go

```diff
@@ -75,11 +80,13 @@ func NewHook(
 	mintKeeper mintkeeper.Keeper,
 	distrKeeper distrkeeper.Keeper,
 	govKeeper govkeeper.Keeper,
+	groupKeeper groupkeeper.Keeper,
 	oracleKeeper oraclekeeper.Keeper,
 	icahostKeeper icahostkeeper.Keeper,
 	clientkeeper clientkeeper.Keeper,
 	connectionkeeper connectionkeeper.Keeper,
 	channelkeeper channelkeeper.Keeper,
+	groupstorekey storetypes.StoreKey,
```

Comment:
groupStoreKey. also, please change for ibc keeper as well.

Reply:
fixed